### PR TITLE
Add some extra redirects & bug fixes.

### PR DESCRIPTION
**Pull request by DFurnes**
### Changes
This pull request re-configures some redirects for Uncle Mike – the homepage ([next.dosomething.org](https://next.dosomething.org)) now sends visitors directly to Sincerely Us, and the "next" campaign list can be viewed at `/campaigns`. It also includes a few bug fixes from #266.

👋

### CODE REVIEW

File: resources/assets/components/LedeBanner/index.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ LedeBanner.propTypes = {
     description: PropTypes.string,
     url: PropTypes.string,
   }).isRequired,
-  experiments: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+  experiments: PropTypes.array, // eslint-disable-line react/forbid-prop-types
```
Comment (deadlybutter) [inline resources/assets/components/LedeBanner/index.js]: Wait, did the data structure change?

Comment (weerd) [inline resources/assets/components/LedeBanner/index.js]: ??

Comment (deadlybutter) [inline resources/assets/components/LedeBanner/index.js]: In the latest dev this is an object, 

screen shot 2017-05-19 at 2 24 45 pm

This prop is getting experiments from [here](https://github.com/DoSomething/phoenix-next/blob/dev/resources/assets/components/Chrome.js#L26), and that comes from [this](https://github.com/DoSomething/phoenix-next/blob/dev/resources/assets/containers/ChromeContainer.js#L26) container which is just grabbing the object

Comment (DFurnes) [inline resources/assets/components/LedeBanner/index.js]: Ah yep, but it gives an array (:interrobang:) if Sixpack is disabled. Updated to accept either.

File: resources/assets/components/LedeBanner/index.js [hunk 2]
```diff
@@ -68,7 +68,7 @@ LedeBanner.propTypes = {
     description: PropTypes.string,
     url: PropTypes.string,
   }).isRequired,
-  experiments: PropTypes.object, // eslint-disable-line react/forbid-prop-types
+  experiments: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
```
Comment (weerd) [inline resources/assets/components/LedeBanner/index.js]: :dancing_men:

**Pull request merged**